docs/workers.md

```diff
@@ -426,6 +426,7 @@ Handles the media repository. It can handle all endpoints starting with:
     ^/_synapse/admin/v1/user/.*/media.*$
     ^/_synapse/admin/v1/media/.*$
     ^/_synapse/admin/v1/quarantine_media/.*$
+    ^/_synapse/admin/v1/users/.*/media$
```

Comment:
(I actually wanted to comment on line 418...)

I think this change is saying that this endpoint *must* be handled by the media repo? "It *can* handle all endpoints start with..." makes it sound like an optional thing.

Reply:
Good spot! Well, the main process can also handle it; but not when media workers are in use, as media repository functionality is typically disabled on the main process in that case.

I've added a line to help clarify things.